ci/generate-markdown/update-library-landing-dox.sh

```diff
@@ -68,6 +68,43 @@ EOT
 
 IFS= mapfile -d $'\0' -t samples_cc < <(git ls-files -z -- "${LIB}/*samples/*_client_samples.cc")
 
+(
+  sed '/<!-- inject-client-list-start -->/q' "${MAIN_DOX}"
+  if [[ ${#samples_cc[@]} -eq 0 ]]; then
+    true
+  elif [[ ${#samples_cc[@]} -eq 1 ]]; then
+    sample_cc="${samples_cc[0]}"
+    client_name="$(sed -n '/main-dox-marker: / s;// main-dox-marker: \(.*\);\1;p' "${sample_cc}")"
+    cat <<_EOF_
+The main class in this library is
+[\`${client_name}\`](@ref google::cloud::${client_name}),
+all RPCs are exposed as member functions in this class. Other classes provide
```

Comment:
Just below we say, "as member functions _of_ this class".

Reply:
Used "*of* this class" in both places